src/Widgets/DataSearchCountWidget/DataSearchCountWidgetComponent.tsx

```diff
@@ -0,0 +1,42 @@
+import { connect, ContentTag, provideComponent, useData } from 'scrivito'
+import {
+  DataSearchCountWidget,
+  DataSearchCountWidgetInstance,
+} from './DataSearchCountWidgetClass'
+import { EditorNote } from '../../Components/EditorNote'
+
+provideComponent(DataSearchCountWidget, ({ widget }) => (
+  <h1 className="h3 text-center">
+    <TotalCountSummary widget={widget} />
+  </h1>
+))
+
+const TotalCountSummary = connect(
```

Comment:
* [ ] FYI "total count summary" looks like a very elaborate name for... "count"?